streetwise/api/images.py

```diff
@@ -3,17 +3,27 @@
 http://flask-restplus.readthedocs.io
 """
 
-from flask import request
+from flask import request, g
 from flask_restplus import Resource, fields
 from sqlalchemy.sql.expression import func
 
 from ..models import Image
 from . import api_rest
 
+from .helper.image_tracker import (
+    init_image_counter, least_displayed_images
+)
+
 ns = api_rest.namespace('image',
     description = 'Image operations'
 )
 
+# How many images to randomly select before counting
+IMAGES_RANDOM_WALK = 50
```

Comment:
Why 10? Why not 50 or even 100? A higher number leads to a better distribution, but slower performance.